README.md

```diff
@@ -95,7 +96,7 @@ Install required dependencies above
 Check that you have SDK Man installed properly (otherwise follow instructions on the skdman install page).
 ```
 $ sdk version
-SDKMAN 3.1.0
+SDKMAN 5.13.2
```

Comment:
For what it's worth, this instruction is just meant to show that a version gets returned. I don't want us to have to update this every time SDKMAN releases a new version.